data/io.elementary.files.appdata.xml.in.in

```diff
@@ -58,6 +58,29 @@
     <content_attribute id="money-gambling">none</content_attribute>
   </content_rating>
   <releases>
+    <release version="Next" date="Future" urgency="medium">
+      <description>
+        <p>New features:</p>
+        <ul>
+          <li>Initial git plugin support</li>
+          <li>Check hicolor theme if no elementary icon before falling back to default</li>
```

Comment:
What exactly does this mean for users?

Reply:
It prevents missing icons in some cases.